### PR TITLE
[GSoC] Fix state space of Wiener Process

### PR DESCRIPTION
**Pull request by Smit-create**
#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by Smit-create** [comment]
@czgdp1807 Please merge this when the tests pass

**Comment by Smit-create** [comment]
Failing job on Travis: https://travis-ci.org/github/sympy/sympy/jobs/716238524

File: sympy/stats/stochastic_process_types.py [hunk 1]
```diff
@@ -1575,6 +1576,10 @@ def lamda(self):
     def gamma(self):
         return self.args[2]
 
+    @property
+    def state_space(self):
+        return _set_converter(Interval(0, oo))
+
```
Comment (czgdp1807) [inline sympy/stats/stochastic_process_types.py:1582]: Add test for this.

File: sympy/stats/stochastic_process_types.py [hunk 2]
```diff
@@ -1505,10 +1501,15 @@ class WienerProcess(CountingProcess):
     .. [2] https://en.wikipedia.org/wiki/Wiener_process
 
     """
+
```
Comment (czgdp1807) [inline sympy/stats/stochastic_process_types.py:1504]:
```suggestion
```

**Pull request merged**